src/components/ToyForm.js

```diff
@@ -22,13 +22,15 @@ const ToyForm = ({ toy }) => {
     event.preventDefault();
     dispatch(createToy(form, toy));
     setForm({
-      ...form,
+      form,
```

Comment:
Please see my original comments (and see above as well).  Passing form here is going to create an object:
```
{ form: { name: '', description: '', imageUrl: '' }, name: '', description: '', imageUrl: '' }
```

..then progressively get larger each time.

Reply:
https://jsfiddle.net/rh9txdjn/

Reply:
Hey Shawn! I went ahead and updated this form. I hope you like the changes. I also got the `imageUrl `to work for the front-end and the back-end. Is that code okay? Thanks!